stable/nats/values.yaml

```diff
@@ -15,7 +15,7 @@ image:
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
   ##
   # pullSecrets:
-  #   - myRegistrKeySecretName
+  #   - name: myRegistrKeySecretName
```

Comment:
please add the option to the README too

Reply:
Funny enough, it was already present in the README, but the actual templated code was missing.